### ISSUE TITLE
It is impossible to define exclude twice - second overrides the first

### ISSUE THREAD

**Issue by matkoniecz** [bbatsov/rubocop#3208]
Excluding file both in .rubocop_todo.yml .rubocop_todo.yml results in ignored exclusion from  .rubocop_todo.yml.

## Expected behavior

With

.rubocop.yml

```
inherit_from: .rubocop_todo.yml 

Metrics/AbcSize:
  Exclude:
    - 'b.rb'

Metrics/MethodLength:
  Exclude:
    - 'b.rb'
```

and .rubocop_todo.yml

```
Metrics/AbcSize:
  Exclude:
    - 'a.rb'

Metrics/MethodLength:
  Exclude:
    - 'a.rb'
```

both a.rb and b.rb will be excluded.

## Actual behavior

```
Inspecting 2 files
.C

Offenses:

a.rb:1:1: C: Metrics/AbcSize: Assignment Branch Condition size for test is too high. [16/15]  (http://c2.com/cgi/wiki?AbcMetric)
def test
^^^
a.rb:1:1: C: Metrics/MethodLength: Method has too many lines. [16/10]  (https://github.com/bbatsov/ruby-style-guide#short-methods)
def test ...
^^^^^^^^

2 files inspected, 2 offenses detected
```

## Steps to reproduce the problem
Create rubocop config as above, create a.rb and b.rb with

```
def test
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
  puts 'a'
end

````

## RuboCop version

Include the output of `rubocop -V`:

```
$ rubocop -V
0.40.0 (using Parser 2.3.1.2, running on ruby 2.2.4 i386-linux-gnu)
```

**Comment by jonas054** [comment]
Arrays are overridden, not extended, by derived configuration. The thinking behind this is that it must be possible to get rid of elements in base configuration that you don't want.

But maybe we can come up with a pretty simple solution to this problem. We introduce a "keyword" called `super` that copies all elements in an inherited parameter, from base to derived, when it appears in an array. So, in `.rubocop.yml`
```yaml
Metrics/AbcSize:
  Exclude:
    - super
    - 'b.rb'
```
would exclude both `a.rb` and `b.rb`, since `.rubocop_todo.yml` is the base configuration

**Comment by Drenmi** [comment]
Hm. I actually like the simplicity of it. No unpredictable things happen and, most importantly, you only need to look in a single place to find the configuration for a cop. This behaviour is also documented in the manual now.

**Comment by jonas054** [comment]
I'm also a big fan of simplicity, @Drenmi. :smile: Since there's no massive support for inheritance of parameter arrays, I'm closing the issue.
